akka-remote/src/main/scala/akka/remote/Remoting.scala

```diff
@@ -409,11 +411,9 @@ private[remote] class EndpointManager(conf: Config, log: LoggingAdapter) extends
   // Mapping between transports and the local addresses they listen to
   var transportMapping: Map[Address, AkkaProtocolTransport] = Map()
 
-  def retryGateEnabled = settings.RetryGateClosedFor > Duration.Zero
-  val pruneInterval: FiniteDuration = if (retryGateEnabled) settings.RetryGateClosedFor * 2 else Duration.Zero
-  val pruneTimerCancellable: Option[Cancellable] = if (retryGateEnabled)
-    Some(context.system.scheduler.schedule(pruneInterval, pruneInterval, self, Prune))
-  else None
+  val pruneInterval: FiniteDuration = settings.RetryGateClosedFor * 2
+  val pruneTimerCancellable: Cancellable =
+    context.system.scheduler.schedule(pruneInterval, pruneInterval, self, Prune)
```

Comment:
It turns out its not:

    val RetryGateClosedFor: FiniteDuration = {
      config.getMillisDuration("akka.remote.retry-gate-closed-for")
    } requiring (_ >= Duration.Zero, "retry-gate-closed-for must be >= 0")

Equality must go.